workshop/05-deploy.md

```diff
@@ -0,0 +1,198 @@
+
+
+## Create Node.js Cloud Foundry app:
+
+In this step, we need to go into the Bluemix site and from the Cloud Foundry section of their catalog, we will choose to create a Node.js Runtime App. Choosing the default options will do for now.
+
+**The catalog:**
+
+![Bluemix - Cloud Foundry Apps](assets/cloud-foundry-apps.png)
+
+**Create the Node.js app**
+
+![Bluemix - Create Node.js app](assets/create-nodejs-app.png)
+
+_Note: take notice of the org and space you created this app in. It is likely you only have one of each, but it is not uncommon to have more than one of one or the other._
+
+The page that is presented after creating your Node.js app shows your app starting up. We can now switch back to the terminal and our editor.
+
+On this page, you will also see some instructions for using a sample app and getting started. We are going to skip that in favor of the following steps:
+
+## Create our `manifest.yml` file
+
+Let's go into the root of our Node.js application and create a `manifest.yml` file and add the following information, changing "name" to the name you chose when creating the Cloud Foundry app.
+
+It should look something like so:
+
+```
+---
+applications:
+ - name: band-aid
+   random-route: true
+   memory: 256M
+```
+
+## Get Cloud Foundry
+
+At this point, we are going to begin using the Cloud Foundry cli. It is required that we install this toolkit before going further.
+
+The most common way of installing Cloud Foundry on a Mac is to use Homebrew:
+
+```
+$ brew tap cloudfoundry/tap
+$ brew install cf-cli
+```
+
+To find instructions for other platforms, see [this page](https://github.com/cloudfoundry/cli#downloads)
+
+## Login `cf login`
+
+The first thing we need to do is log in. Doing so, it will prompt us for email, password, organization and space.
+
+In the terminal, let's run `cf login`:
+
+```
+➜  band-aid git:(master) ✗ cf login
+API endpoint: https://api.ng.bluemix.net
+
+Email> joesepi@ibm.com
+
+Password>
+Authenticating...
+OK
+
+Select an org (or press enter to skip):
+1. StrongLoop Evangelists Org
+2. Some Total
+
+Org> 2
+Targeted org Some Total
+
+Select a space (or press enter to skip):
+1. dev-unkempt
+2. BMLabs-PROD
+3. dev
+4. eventquarry-dev
+5. Band-Aid
+
+Space> 5
+Targeted space Band-Aid
+
+
+
+API endpoint:   https://api.ng.bluemix.net (API version: 2.54.0)
+User:           joesepi@ibm.com
+Org:            Some Total
+Space:          Band-Aid
+```
+
+Looks like we were successful.
+
+## Set Region `cf api`
+
+Now we need to set the region for Bluemix. There are only 3, so we just need to match it up with the region our Org/Space is using in Bluemix. Here are the three options:
+
+|API endpoint                   |Region
+|---                            |---
+|https://api.ng.bluemix.net	|US South
+|https://api.eu-gb.bluemix.net	|United Kingdom
+|https://api.au-syd.bluemix.net	|Sydney
+
+The region that I am using is US South, so you will see that set below (and also in the output above).
```

Comment:
Yeah, with this one, I struggled. But the fact is that I am using US South, but someone else doing this workshop may not be, so I will leave this one as is. Thanks!